common/djangoapps/student/tests/test_views.py

```diff
@@ -316,3 +318,27 @@ def test_sharing_icons_for_future_course(self, set_marketing, set_social_sharing
         response = self.client.get(reverse('dashboard'))
         self.assertEqual('Share on Twitter' in response.content, set_marketing or set_social_sharing)
         self.assertEqual('Share on Facebook' in response.content, set_marketing or set_social_sharing)
+
+    @patch.dict("django.conf.settings.FEATURES", {'ENABLE_PREREQUISITE_COURSES': True})
+    def test_pre_requisites_appear_on_dashboard(self):
+        """
+        When a course has a prerequisite, the dashboard should display the prerequisite.
+        If we remove the prerequisite and access the dashboard again, the prerequisite
+        should not appear.
+        """
+        self.pre_requisite_course = CourseFactory.create(org='edx', number='999', display_name='Pre requisite Course')
+        self.course = CourseFactory.create(
+            org='edx',
+            number='998',
+            display_name='Test Course',
+            pre_requisite_courses=[unicode(self.pre_requisite_course)]
```

Comment:
Maybe throw the `.id` onto the send of `self.pre_requisite_course` to make it a little more explicit (even though the repr of a course is the same as the repr of it's id)?